### PR TITLE
[native] Add e2e tests for normalize scalar function

### PR DESCRIPTION
**Pull request by pdabre12**
Add e2e tests for normalize Presto scalar function

### CODE REVIEW

**Comment by pdabre12** [comment]
@aditi-pandit PTAL, thanks.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java [hunk 1]
```diff
@@ -830,6 +830,15 @@ public void testStringFunctions()
 
         // Reverse
         assertQuery("SELECT comment, reverse(comment) FROM orders");
+
+        // Normalize
+        assertQuery("SELECT normalize(null)");
```
Comment (aditi-pandit) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java:835]: Thanks @pdabre12 

All the queries from 835 - 839 will be executed at the co-ordinator as they use constants. Please add more queries using columns like those on line 840/841. But orders table is very large. Maybe use a field from nations or regions table instead. Or another option is to create a table and insert interesting data into it in the test itself.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java [hunk 2]
```diff
@@ -830,6 +830,27 @@ public void testStringFunctions()
...
+        // Normalize
+        String tmpTableName = generateRandomTableName();
+        try {
+            getQueryRunner().execute(String.format("CREATE TABLE %s (c0 VARCHAR)", tmpTableName));
+            getQueryRunner().execute(String.format("INSERT INTO %s VALUES " +
+                    "('\\u3231\\u3327\\u3326\\u2162\\u3231\\u3327\\u3326\\u2162\\u3231\\u3327\\u3326\\u2162'), " +
+                    "('\\xEF\\xBE\\x8'), " +
+                    "('sch\\u00f6n') ", tmpTableName));
+
+            assertQuery("SELECT normalize(null)");
```
Comment (aditi-pandit) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java:843]: Queries on line 843 and 844 are not needed as they evaluate on the co-ordinator.

**Review by aditi-pandit — COMMENTED** [review]
@pdabre12 : Minor comment.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java (hunk 2 above)
Comment (pdabre12) [reply presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java]: Made the changes, thanks.

**Review by aditi-pandit — APPROVED** [review]
Thanks @pdabre12

**Pull request merged**